calico-enterprise/networking/gateway-api.mdx

```diff
@@ -0,0 +1,418 @@
+tlsroutes                                                                           gateway.networking.k8s.io/v1alpha2          true         TLSRoute
+udproutes                                                                           gateway.networking.k8s.io/v1alpha2          true         UDPRoute
+```
+
+And also that there is a GatewayClass resource corresponding to the Envoy Gateway implementation included in {{prodname}}:
+
+```bash
+kubectl get gatewayclass -o yaml | yq r - 'items[0].spec'
+```
+
+```
+controllerName: gateway.envoyproxy.io/gatewayclass-controller
+parametersRef:
+  group: gateway.envoyproxy.io
+  kind: EnvoyProxy
+  name: envoy-proxy-config
+  namespace: tigera-gateway
+```
+
+### Use the Gateway API
+
+Now you can use the Gateway API in any way that the API supports.  When configuring a Gateway resource, be sure to specify `gatewayClassName: tigera-gateway-class`.
+
+By way of a simple example:
+
+- Create echo servers in namespaces `ns1` and `ns2`:
+
+  ```bash
+  kubectl apply -f - <<EOF
+  apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: ns1
+  ---
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: echoserver-ns1
+    namespace: ns1
+    labels:
+      app: echoserver
+  spec:
+    containers:
+    - image: ealen/echo-server:latest
+      imagePullPolicy: IfNotPresent
+      name: echoserver
+      ports:
+      - containerPort: 80
+      env:
+      - name: PORT
+        value: "80"
+  ---
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: echoserver
+    namespace: ns1
+  spec:
+    ports:
+      - port: 80
+        targetPort: 80
+        protocol: TCP
+    type: ClusterIP
+    selector:
+      app: echoserver
+  ---
+  apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: ns2
+  ---
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: echoserver-ns2
+    namespace: ns2
+    labels:
+      app: echoserver
+  spec:
+    containers:
+    - image: ealen/echo-server:latest
+      imagePullPolicy: IfNotPresent
+      name: echoserver
+      ports:
+      - containerPort: 80
+      env:
+      - name: PORT
+        value: "80"
+  ---
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: echoserver
+    namespace: ns2
+  spec:
+    ports:
+      - port: 80
+        targetPort: 80
+        protocol: TCP
+    type: ClusterIP
+    selector:
+      app: echoserver
+  EOF
+  ```
+
+- Create a client pod that we can test from:
+
+  ```bash
+  kubectl apply -f - <<EOF
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: test-client
+  spec:
+    containers:
+    - args:
+      - "3600"
+      command:
+      - /bin/sleep
+      image: ubuntu
+      imagePullPolicy: Always
+      name: c1
+    terminationGracePeriodSeconds: 0
+  EOF
+  ```
+
+- Once that pod is running, install `curl` in it:
+
+  ```bash
+  kubectl exec -it test-client -- apt-get -y update
+  kubectl exec -it test-client -- apt-get -y install curl
+  ```
```

Comment:
Could use an image that already has curl installed. I typically use `wbitt/network-multitool` for that kind of stuff (not sure if there are better alternatives).